clang/unittests/Analysis/FlowSensitive/TransferTest.cpp

```diff
@@ -3893,6 +3893,66 @@ TEST(TransferTest, BooleanInequality) {
       });
 }
 
+TEST(TransferTest, PointerEquality) {
```

Comment:
I think it would be great to add a test case with union members to document the limitation.

Reply:
Good point -- done.